TOC.md

```diff
@@ -2,187 +2,204 @@
-      - [Binlog Event Filtering](tools/dm/data-synchronization-features.md#binlog-event-filtering)
-      - [Column Mapping](tools/dm/data-synchronization-features.md#column-mapping)
-      - [Synchronization Delay Monitoring](tools/dm/data-synchronization-features.md#synchronization-delay-monitoring)
-      + Sharding Support
-        - [Introduction](tools/dm/shard-merge.md)
-        - [Restrictions](tools/dm/shard-merge.md#restrictions)
-        - [Handle Sharding DDL Locks Manually](tools/dm/manually-handling-sharding-ddl-locks.md)
-    + Usage Scenarios
-      - [Simple Scenario](tools/dm/simple-synchronization-scenario.md)
-      - [Shard Merge Scenario](tools/dm/shard-merge-scenario.md)
-    - [Deploy a DM Cluster](tools/dm/deployment.md)
-    - [Migrate Data Using DM](tools/dm/practice.md)
-    + Configure
-      - [Overview](tools/dm/dm-configuration-file-overview.md)
-      - [Task Configuration](tools/dm/task-configuration-file-intro.md)
-    - [Monitor](tools/dm/monitor.md)
-    - [Manage the Task](tools/dm/manage-task.md)
-    - [Cluster Operations](tools/dm/cluster-operations.md)
-    - [Upgrade](tools/dm/dm-upgrade.md)
-    - [Troubleshoot](tools/dm/troubleshooting.md)
-  + TiDB-Lightning
-    - [Overview](tools/lightning/overview-architecture.md)
-    - [Deployment](tools/lightning/deployment.md)
-    - [Checkpoints](tools/lightning/checkpoints.md)
-    - [Table Filter](tools/lightning/filter.md)
-    - [CSV Support](tools/lightning/csv.md)
-    - [Monitor](tools/lightning/monitor.md)
-    - [Troubleshooting](tools/lightning/errors.md)
-    - [FAQs](tools/lightning/faq.md)
-  - [TiDB-Binlog](tools/tidb-binlog-cluster.md)
-  - [PD Control](tools/pd-control.md)
-  - [PD Recover](tools/pd-recover.md)
-  - [TiKV Control](https://github.com/tikv/tikv/blob/master/docs/tools/tikv-control.md)
-  - [TiDB Controller](tools/tidb-controller.md)
-  - [Download](tools/download.md)
-+ [TiKV Documentation](https://github.com/tikv/tikv/wiki)
-+ TiSpark Documentation
-  - [Quick Start Guide](tispark/tispark-quick-start-guide.md)
-  - [User Guide](tispark/tispark-user-guide.md)
-- [Frequently Asked Questions (FAQ)](FAQ.md)
-- [TiDB Best Practices](https://pingcap.com/blog/2017-07-24-tidbbestpractice/)
+    + Scale
+      - [Scale a TiDB Cluster](op-guide/horizontal-scale.md)
+    + Common Ansible Operations
+      - [Overview](op-guide/ansible-operation.md)
+      - [Scale](op-guide/ansible-deployment-scale.md)
+      - [Modify Component Configuration](op-guide/ansible-deployment-rolling-update.md#modify-component-configuration)
+      - [Upgrade Component Version](op-guide/ansible-deployment-rolling-update.md#upgrade-the-component-version)
+  - Troubleshoot
+    - [Common Errors](sql/error.md)
+    - [Troubleshoot Cluster Setup](trouble-shooting.md)
+    - [Troubleshoot Migrations](tools/dm/troubleshooting.md)
+    - [Troubleshooting TiDB-Lightning Imports](tools/lightning/errors.md)
```

Comment:
We also have a troubleshooting file for DM: https://github.com/pingcap/docs/blob/master/tools/dm/troubleshooting.md